buildinfo_test.go

```diff
@@ -280,19 +287,32 @@ func TestBuildAppend(t *testing.T) {
 
 func TestBuildAddDependencies(t *testing.T) {
 	initArtifactoryTest(t)
+	createJfrogHomeConfig(t, true)
 	// Clean old build tests if exists
 	inttestutils.DeleteBuild(serverDetails.ArtifactoryUrl, tests.RtBuildName1, artHttpDetails)
-
+	buildAddDepsRemoteSpec, err := tests.CreateSpec(tests.BuildAddDepsRemoteSpec)
+	assert.NoError(t, err)
+	buildAddDepsDoubleRemoteSpec, err := tests.CreateSpec(tests.BuildAddDepsDoubleRemoteSpec)
+	assert.NoError(t, err)
+	artifactoryCli.Exec("upload", "testdata/a/*", tests.RtRepo1, "--flat=false")
 	allFiles := []string{"a1.in", "a2.in", "a3.in", "b1.in", "b2.in", "b3.in", "c1.in", "c2.in", "c3.in"}
 	var badTests = []buildAddDepsBuildInfoTestParams{
 		{description: "'rt bad' simple cli", commandArgs: []string{"testdata/a/*"}, expectedDependencies: allFiles},
```

Comment:
Let's add the following comment above this line.

// Collect the dependencies from the local file system (the --from-rt option is not used).